components/alarm_control_panel/template.rst

```diff
@@ -62,9 +64,19 @@ State Flow:
   b. ``arming_..._time`` is 0 or after the delay the state is ``ARMED_...``
 
 3. When the alarm is tripped by a sensor state changing to ``on`` or ``alarm_control_panel_pending_action`` invoked
+  1. If sensor_type is set to ``delayed``:
```

Comment:
```suggestion
  1. If sensor_type is set to ``delayed``:
  
```

Reply:
Thanks for the suggestions. It passed all the checks after making the edits.